admin/src/css/theme.less

```diff
@@ -1,25 +1,47 @@
-.body,
-.header {
-  padding: 20px;
+@nav-active-color: #38464d;
+body {
+  max-width: @admin-content-width;
+  margin: 0 auto;
+  font-size: @font-medium;
```

Comment:
I don't think this will work because `@font-medium` is set to `1rem` which means take the base font size and use that, but we haven't set a base font size so it'll use the browser default of 14px. We want 16px.

In webapp we have this rule as well:

```css
html {
  font-size: 16px;
}
```

That sets the base font size for the whole doc meaning `rem`s work as expected!